Add `deepKeys()` (#94)
sindresorhus/dot-prop: readme.md

//=> '🍄 The princess is in another castle!'
```

### deepKeys(object)

> Returns an array that contains every path in `object`. Plain objects are iterated deeper and are not themselves yielded. Useful to help flatten an object for an API that only accepts key-value pairs or for a tagged template literal.

Reviewer: ```suggestion
Returns an array that contains every path in the given. Plain objects are deeply recursed and are not themselves yielded.

This can be useful to help flatten an object for an API that only accepts key-value pairs or for a tagged template literal.
```

Reviewer: And I don't think it should mention "yielding".
